guides/v2.2/rest/tutorials/configurable-product/config-product-intro.md

```diff
@@ -14,27 +14,22 @@ functional_areas:
   - Integration
```

Comment:
GitHub won't let me comment above this line, but the `return_to:` front matter tag (line 8) points to REST API Reference instead of REST Tutorials like in `rest/tutorials/orders/order-intro.html`. I'm not sure which is correct, but both should return to the same topic.

Reply:
This particular file was the only one in the configurable product tutorial to have the incorrect `return_to` values. Fixed.